src/components/Table.js

```diff
@@ -1,74 +1,114 @@
+        stats: [
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: "" },
+          { status: "" }
+        ]
+      },
+      GYM_HABIT: {
+        name: "Спортзал",
+        measuringValue: "3р/нед",
+        stats: [
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.DONE },
+          { status: Status.NEUTRAL },
+          { status: Status.NOT_SPECIFIED },
+          { status: Status.NOT_SPECIFIED }
+        ]
+      },
+      EARLY_RISE_HABIT: {
+        name: "Ранний подъем",
+        measuringValue: "6:30",
+        stats: [
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.NOT_SPECIFIED },
+          { status: Status.NOT_SPECIFIED }
+        ]
+      }
+    };
   }
 
-  function getDayFromMonday(num) {
-    const date = new Date();
-    return new Date(date.setDate(monday.getDate() + num));
-  }
-
-  const monday = getMonday(new Date());
-  const tuesday = getDayFromMonday(1);
-  const wednesday = getDayFromMonday(2);
-  const thursday = getDayFromMonday(3);
-  const friday = getDayFromMonday(4);
-  const saturday = getDayFromMonday(5);
-  const sunday = getDayFromMonday(6);
+  handleClickCell = e => {
+    const test = e.target;
+    console.log(test);
+  };
 
-  const listOfHabitsNames = HABITS.map(habit => <td>{habit.name}</td>);
+  render() {
+    const HABITS = [
+      this.state.READING_HABIT,
+      this.state.GYM_HABIT,
+      this.state.EARLY_RISE_HABIT
+    ];
+    const listOfHabitsNames = HABITS.map(habit => <td>{habit.name}</td>);
 
-  const listOfMeasureValue = HABITS.map(habit => (
-    <td>{habit.measuringValue}</td>
-  ));
+    const listOfMeasureValue = HABITS.map(habit => (
+      <td>{habit.measuringValue}</td>
+    ));
 
-  return (
-    <div className="table-wrapper container">
-      <table className="table table-bordered">
-        <tr>
-          <th>Навык</th>
-          {listOfHabitsNames}
-          <td>Потенциал дня</td>
-        </tr>
-        <tr>
-          <th>Норма</th>
-          {listOfMeasureValue}
-          <td>1...10</td>
-        </tr>
-        {[monday, tuesday, wednesday, thursday, friday, saturday, sunday].map(
-          (day, dayOrderNumber) => (
+    return (
+      <div className="table-wrapper container">
+        <table className="table table-bordered">
+          <tr>
+            <th>Навык</th>
+            {listOfHabitsNames}
+            <td>Потенциал дня</td>
+          </tr>
+          <tr>
+            <th>Норма</th>
+            {listOfMeasureValue}
+            <td>1...10</td>
+          </tr>
+          {DateUtils.getWeekDays(new Date()).map((day, dayOrderNumber) => (
```

Comment:
You don't need to compute the list of weekDays every time Table renders. You can generate the list of weekDays in App component and pass it through `weekDays` prop

```js
const App = () => {
  const weekDays = DateUtils.getWeekDays(new Date());
  return (
    <Table weekDays={weekDays} />
  );
};
```

Reply:
Yes, you are right. Fixed

Reply:
Yes, you are right. Fixed